kolibri/core/content/utils/paths.py

```diff
@@ -140,6 +140,9 @@ def get_content_storage_file_url(filename, baseurl=None):
             kwargs={"zipped_filename": filename, "embedded_filepath": ""},
         )
     else:
-        return "{}{}/{}/{}".format(
-            get_content_storage_url(baseurl), filename[0], filename[1], filename
+        return "/{}{}/{}/{}".format(
```

Comment:
nice catch, I would just remove the lstrip function as it's not needed.

Reply:
if lstrip is left out, wouldnt that affect the case where there is no prefix, ie "/" ending up with //5/a/5afdfe.mp4

Reply:
it should not affect because the prefix is cleaned at https://github.com/learningequality/kolibri/blob/develop/kolibri/utils/options.py#L152